component/attr_cache/attr_cache.go

```diff
@@ -715,20 +615,29 @@ func (ac *AttrCache) RenameFile(options internal.RenameFileOptions) error {
 	err := ac.NextComponent().RenameFile(options)
 	if err == nil {
 		renameTime := time.Now()
+		ac.cacheLock.RLock()
+		defer ac.cacheLock.RUnlock()
+		// TODO: Can we just copy over the attributes from the source to the destination so we don't have to invalidate?
+
+		toBeDeleted, getErr := ac.cacheMap.get(options.Src)
+		if getErr != nil {
+			log.Warn("AttrCache::RenameFile : %s", err)
+		} else {
+			ac.moveAttrCachedItem(toBeDeleted, options.Src, options.Dst, time.Now())
+		}
+
+		toBeInvalid, getErr := ac.cacheMap.get(options.Dst)
+		if getErr != nil {
+			log.Err("AttrCache::RenameFile : %s", err)
+		} else {
+			toBeInvalid.invalidate()
+		}
```

Comment:
```suggestion
```
If we're moving the item above, why are we invalidating it here?
I think we can skip this part entirely.